store/copr/coprocessor.go

```diff
@@ -272,6 +272,10 @@ type copTask struct {
 
 	// timeout value for one kv readonly request
 	tikvClientReadTimeout uint64
+	// timeout value for one kv readonly reqeust
+	tidbKvReadTimeout uint64
+	// firstReadType is used to indicate the type of first read when retrying.
+	firstReadType string
```

Comment:
When is the `firstReadType` initialized to a non-empty value for coprocessor tasks?

Reply:
[here](https://github.com/pingcap/tidb/pull/47003/files#diff-23eb1043649369d5956b123b173b799ded8c6b7cbf4519bb32cb3caa01928d75R1091), and `req.ReadType` will be initialized by replica selector.